yb-voyager/src/dbzm/valueConverter.go

```diff
@@ -86,7 +92,12 @@ func (conv *DebeziumValueConverter) ConvertRow(tableName string, columnNames []s
 		}
 		columnValues[i] = transformedValue
 	}
-	return strings.Join(columnValues, "\t"), nil
+	var buffer bytes.Buffer
```

Comment:
Consider having a `bytes.Buffer` instance as a member of `DebeziumValueConverter`.

Reuse the same buffer (after calling `conv.buf.Reset()`) across multiple runs of the `ConvertRow()`.